front-end/packages/robustar/src/components/prediction-viewer/Visualizer.vue

```diff
@@ -172,8 +172,8 @@ export default {
           this.proposedEditUrl = '';
           return;
         }
-        const proposedPath = res.data.data;
-        this.proposedEditUrl = `${configs.imagePathServerUrl}?${configs.imagePathParamName}=${proposedPath}`;
+        const { base64 } = res.data.data;
+        this.proposedEditUrl = base64;
```

Comment:
This is passed to `ProposedEditView` component right away. Does it work if we don't make modification to `ProposedEditView`?
Also, we may want to change the variable name here because we are no longer using image url.